doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -99,12 +99,22 @@ Keywords
   The following character sequences are reserved keywords that cannot be
   used as identifiers::
 
-    _ Axiom CoFixpoint Definition Fixpoint Hypothesis IF Parameter Prop
-    SProp Set Theorem Type Variable as at by cofix discriminated else
-    end exists exists2 fix for forall fun if in lazymatch let match
-    multimatch return then using where with
+    _ Prop SProp Set Type as at cofix else end fix for forall fun if
+    in let match return then where with
 
-  Note that plugins may define additional keywords when they are loaded.
+  Note however that plugins may define additional keywords when they
```

Comment:
I would not add "however" here.  "In addition" would be good, but having both "In addition" and "additional" in the sentence seems sub par.

Definition of "however": "used to introduce a statement that contrasts with or seems to contradict something that has been said previously".  This is not a contrast or contradiction, right?

(I think it may be excruciating for everyone to frequently discuss wording and style at this level.  We should think about our process for making these adjustments as we improve the doc.)

Reply:
No worry, I like discussing words. The problem is more that my English is not very good.

Would something like this be better?

"The following character sequences are reserved keywords that cannot be used as identifiers::                                                                               
                                                                                                      
    _ Prop SProp Set Type as at cofix else end fix for forall fun if in let match return then where with

Note that the set of reserved keywords is extensible though: plugins or libraries can introduce new keywords when loaded." be better?

Reply:
IMHO you could just change "keywords" to "reserved keywords" and "plugins" to "plugins and libraries" in the original sentence.  "the set of" and "though" are not necessary, they don't add anything.  "extensible" doesn't say anything that "introduce" or "define" don't convey here.  "when loaded" is correct English but is perhaps a bit terse.

Reply:
> No worry, I like discussing words. The problem is more that my English is not very good.

If you like, I'd be happy to point out mistakes or possible improvements from time to time.